subgraphs/_reference_/src/sdk/protocols/config.ts

```diff
@@ -0,0 +1,45 @@
+import { BigDecimal, BigInt } from "@graphprotocol/graph-ts";
+import { Token } from "../../../generated/schema";
+import { Versions } from "../../../../../deployment/context/interface";
+
+export interface ProtocolConfigurer {
+  getID(): string;
+  getName(): string;
+  getSlug(): string;
+  getVersions(): Versions;
+}
+
+export class ProtocolConfig implements ProtocolConfigurer {
+  id: string;
+  name: string;
+  slug: string;
+  versions: Versions;
+
+  constructor(id: string, name: string, slug: string, versions: Versions) {
```

Comment:
I'm surprised this line didn't get formatted...

Reply:
It seems to break down into multiple lines when the number of arguments is >=5 or the line is longer than 81 chars. So its right at the limit 😅